tensorflow/core/kernels/rnn/gru_ops.cc

```diff
@@ -121,6 +121,11 @@ class GRUCellBlockOp : public OpKernel {
         ctx, ctx->allocate_output("c", TensorShape({batch_size, cell_size}),
                                   &c_tensor));
 
+    Tensor* g_tensor = nullptr;
+    OP_REQUIRES_OK(
+        ctx, ctx->allocate_output("g", TensorShape({batch_size, cell_size}),
```

Comment:
IIUC, you're creating a new tensor and allocating memory here. I don't think that's needed or would fix the issue. I would imagine you'd have to add checks to the existing Tensors here. 

Please take a look at this PR, which is fixing something similar: https://github.com/tensorflow/tensorflow/pull/59477